### PR TITLE
update name_scope example code

### PR DESCRIPTION
**Pull request by luotao1**
### PR types

Function optimization
### PR changes

Docs
### Describe

modify docs of name_scope

(1)1.8API list：paddle.fluid.name_scope 
(2)2.0_main_alias：paddle.static.name_scope 
(3) 组长建议：新API名称：暂无,文档和示例待更新 
image

### CODE REVIEW

File: python/paddle/fluid/framework.py [hunk 1]
```diff
@@ -506,33 +506,34 @@ def name_scope(prefix=None):
...
              b = a + 1
-             with fluid.name_scope("s2"):
+             with paddle.static.name_scope("s2"):
                 c = b * 1
-             with fluid.name_scope("s3"):
+             with paddle.static.name_scope("s3"):
                 d = c / 1
-          with fluid.name_scope("s1"):
+          with paddle.static.name_scope("s1"):
                 f = fluid.layers.pow(d, 2.0)
```
Comment (zhangting2020) [inline python/paddle/fluid/framework.py]: fluid.layers需要替换为paddle.xx接口

Comment (luotao1) [reply python/paddle/fluid/framework.py]: done

**Review by zhangting2020 — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM~

**Review by Xreki — APPROVED** [review]

**Pull request merged**